java/src/generated/java/io/cucumber/messages/types/TestCaseStarted.java

```diff
@@ -13,17 +13,20 @@ public final class TestCaseStarted {
     private final Long attempt;
     private final String id;
     private final String testCaseId;
+    private final String workerId;
     private final Timestamp timestamp;
 
     public TestCaseStarted(
         Long attempt,
         String id,
         String testCaseId,
+        String workerId,
```

Comment:
Note that this makes it a breaking change.

Reply:
Oh because of ctor functions in some languages? Would it help to move it to the end?

Reply:
Nope. Not for Java. But that's okay. As long as we follow Semver it's manageable.